azure-iot-provisioning-devicessdk/azure/iot/provisioning/devicessdk/device/registration_client.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module provides an abstract interface representing clients which can communicate with the
+Device Provisioning Service.
+"""
+
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class RegistrationClient(object):
```

Comment:
>RegistrationClient [](start = 6, length = 18)

You call this a Provisioning Client in the sample. Why is it called Registration Client here?

Reply:
changed in sample